### ISSUE TITLE
Git Project on the root of a drive is not seen as a project

### ISSUE THREAD

**Issue by rdemeyer** [atom/tree-view#792]
Hello,

I have a project that is located on the root of a drive, e.g. all my files are in `W:/`, including my `.git` folder. Even if I open this location (`W:/`) as a project, it appears as a folder, not a project, and, as a consequence, git-related coloring (ex: modified files in orange) is not applied (while git commands can be executed through the editor).

Is it possible to overcome this behavior?

Thanks,

RD

**Comment by rdemeyer** [comment]
Hi,

Is there any problem in my description of this issue?

Thanks

**Comment by lee-dohm** [comment]
No, the description seems sound. We haven't had the chance to reproduce this yet though.

@damieng Can you take a look at this?
